distrobox-init

```diff
@@ -2286,7 +2286,7 @@ if [ -e /usr/lib/systemd/systemd ] || [ -e /lib/systemd/systemd ]; then
 	# Start user Systemd unit, this will attempt until Systemd is ready
 	sh -c "	sleep 1 && while true; do \
 	    systemctl is-system-running | grep -E 'running|degraded' && break; \
-	    echo 'waiting for systemd to come up...\n' && sleep 1; \
+	    printf "waiting for systemd to come up...'\n'" && sleep 1; \
```

Comment:
Problem here is that the printf quotes actually are closing che `sh -c` ones

I think it's a bit easier with echo, maype we can add an rpmlint exception? Because I already have shellcheck quite scritly set and it didn't give me any problems